config_generate.py

```diff
@@ -0,0 +1,221 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
This should maybe go into `LabTools.IO.IOTool`. In this module there was was already a function `create_config_file`, did you see it?

Reply:
I would also rename the function `generate_config_file` or replace `create_config_file` with it. (META-DISCUSSION ABOUT WORKFLOW : this can be done in one PR, typically 3-4 commits, will be easy to review and can be merged quickly)

Reply:
I am not opposed to any file renaming, provided it is done with refactor. I designed this to be executed before first run of LabGUI, and offer user prompts for custom names/files/folders/etc.